### PR TITLE
(docs): Add inline left join examples

### PR DESCRIPTION
**Pull request by maarteNNNN**

### CODE REVIEW

**Comment by seveibar** [comment]
i was doing some searching today on complex joins and this was really helpful!

**Review by naorpeled — APPROVED** [review]

**Comment by naorpeled** [comment]
LGTM!

**Review by igalklebanov — CHANGES_REQUESTED** [review]
Thank you! 💪 

Not sure this is worth the maintenance, I believe it was left out for a reason (@koskimas correct me if I'm wrong). 
Wonder if this could be automated (copy `innerJoin` examples, replace occurences with `leftJoin`).

But anyway:

- Missing generated SQL code blocks for both examples.
- Should be pasted to other query builders with left join methods.

File: src/query-builder/select-query-builder.ts [hunk 1]
```diff
@@ -900,6 +900,34 @@ export class SelectQueryBuilder<DB, TB extends keyof DB, O>
...
+   *
+   * ### Examples
+   *
+   * Simple usage:
+   *
+   * ```ts
+   * await db
+   *   .selectFrom('person')
+   *   .leftJoin('address')
+   *   .on('person.address_id', 'address.id')
```
[the remark this answers is not in the document]
Comment (maarteNNNN) [reply src/query-builder/select-query-builder.ts]: Right... My bad, fixed in c6b706362ca8d912d97a1938dea8c4557799bbc2.

**Comment by maarteNNNN** [comment]
> Not sure this is worth the maintenance

Let me know if this can/will go forward then I'll add the generated SQL and adapt all other query builders with left joins.

**Comment by koskimas** [comment]
Yeah the examples were left out on purpose to reduce copy-pasta. There's a link to the `innerJoin` docs from all other join methods. The API is exactly the same for all join methods. Copy-pasting the examples to all methods doesn't make sense to me. They will diverge.

**Comment by koskimas** [comment]
Should we make it more clear that you can find more documentation behind the link? Something like `For more info see the <a>innerJoin</a> docs`?

**Pull request closed without merging**